packages/template-retail-react-app/app/pages/account/order-history.jsx

```diff
@@ -22,30 +22,85 @@ import {
     Img,
     Skeleton
 } from '@chakra-ui/react'
-import useCustomer from '../../commerce-api/hooks/useCustomer'
+import {useCurrentCustomer} from '../../hooks/use-current-customer'
+import {useCustomerOrders, useProducts} from 'commerce-sdk-react-preview'
 import useNavigation from '../../hooks/use-navigation'
 import {usePageUrls, useSearchParams} from '../../hooks'
-import {useAccountOrders} from './util/order-context'
 import PageActionPlaceHolder from '../../components/page-action-placeholder'
 import Link from '../../components/link'
 import {ChevronRightIcon, ReceiptIcon} from '../../components/icons'
 import Pagination from '../../components/pagination'
+import PropTypes from 'prop-types'
+
+const DEFAULT_PAGINATION_LIMIT = 10
```

Comment:
🤔 should this go in a config file?

Reply:
Yes. I though about it but I saw the content of the constants file already a bit daunting and I didn't want to contribute with adding more values. But it's better to follow the same pattern and keep all the values define in the same place.